app/controllers/admin_controller.rb

```diff
@@ -43,6 +43,22 @@ def demote_basic
     redirect_to '/profile/' + @user.username + '?_=' + Time.now.to_i.to_s
   end
 
+  def reset_user_password
+    if current_user && (current_user.role == 'moderator' || current_user.role == 'admin')
```

Comment:
Hi, doesn't this mean that moderators would be able to reset passwords? If we could just change this to admins only, that'd be great. Thanks!!!

Reply:
I apologize, you are correct. This would allow moderator's to reset a user's password. I will fix this after work and push the changes.